cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -570,4 +571,46 @@ protected void setInitImage(String initImage) {
     protected String getDefaultLogConfigFileName() {
         return "kafkaDefaultLoggingProperties";
     }
+
+    public ServiceAccount generateInitContainerServiceAccount() {
+        return new ServiceAccountBuilder()
+                .withNewMetadata()
+                    .withName(initContainerServiceAccountName(cluster))
+                    .addToLabels("app", "strimzi")
+                    .withNamespace(namespace)
+                .endMetadata()
+            .build();
+    }
+
+
+    /**
+     * Get the name of the kafka service account given the name of the {@code cluster}.
+     */
+    public static String initContainerServiceAccountName(String cluster) {
+        return cluster + "-kafka";
```

Comment:
`topicOperatorServiceAccountName` is using the `NAME_SUFFIX` constant as for the other case. Did you use hardcoded string for any reason, do we want to switch to hardcoded strings everywhere?